src/app/tickets/ticket-form/ticket-form.component.html

```diff
@@ -0,0 +1,97 @@
+              <mat-label>Ticket Title</mat-label>
+              <input type="text" matInput required formControlName="title" />
+            </mat-form-field>
+            <mat-form-field fxFlex="100" appearance="outline">
+              <mat-label>Description</mat-label>
+              <textarea
+                type="text"
+                matInput
+                required
+                formControlName="description"
+              ></textarea>
+            </mat-form-field>
+          </div>
+          <mat-list-item>Service Desk Type Specific fields</mat-list-item>
+          <mat-divider></mat-divider>
+          <div fxLayout="row wrap">
+            <mat-form-field appearance="outline">
+              <mat-label>Example Field 1</mat-label>
+              <input type="text" matInput required />
+            </mat-form-field>
+            <mat-form-field appearance="outline">
+              <mat-label>Example Field 1</mat-label>
+              <input type="text" matInput required />
+            </mat-form-field>
+            <mat-form-field appearance="outline">
+              <mat-label>Example Field 2</mat-label>
+              <input type="text" matInput required />
+            </mat-form-field>
+            <mat-form-field appearance="outline">
+              <mat-label>Example Field 3</mat-label>
+              <input type="text" matInput required />
+            </mat-form-field>
+            <mat-form-field appearance="outline">
+              <mat-label>Example Field 4</mat-label>
+              <input type="text" matInput required />
+            </mat-form-field>
+            <mat-form-field appearance="outline">
+              <mat-label>Example Field 5</mat-label>
+              <input type="text" matInput required />
+            </mat-form-field>
+          </div>
+          <mat-list-item>K's Internal Fields Only</mat-list-item>
+          <mat-divider></mat-divider>
+          <div fxLayout="row wrap">
+            <mat-form-field appearance="outline">
+              <mat-label>Example Field 1</mat-label>
+              <input type="text" matInput required />
+            </mat-form-field>
+            <mat-form-field appearance="outline">
+              <mat-label>Example Field 1</mat-label>
+              <input type="text" matInput required />
+            </mat-form-field>
+            <mat-form-field appearance="outline">
+              <mat-label>Example Field 2</mat-label>
+              <input type="text" matInput required />
+            </mat-form-field>
+          </div>
+        </mat-list>
+      </div>
+    </mat-card-content>
+    <mat-divider></mat-divider>
+    <mat-card-footer>
+      <div fxLayout="row" fxLayoutAlign="end center">
+        <button mat-raised-button (click)="onCancel()">Cancel</button>
+        <button mat-raised-button>Save as Draft</button>
+        <button mat-raised-button color="primary" type="submit">
+          Update
+        </button>
```

Comment:
Should add type="button" to the non-submit buttons - clicking any of the buttons is submitting the from right now.